Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob/UploadBlob.cs

```diff
@@ -10,30 +9,36 @@
 using Azure.Storage.Blobs;
 using Azure.Storage.Blobs.Models;
 using Azure.Storage.Blobs.Specialized;
-
+using System.IO.Compression;
+
 #pragma warning disable CS1591
-
+#pragma warning disable CA2208
+#pragma warning disable IDE0066
```

Comment:
Again, do we really need all those pragmas here? I am somewhat not a big fan of those as they might be hiding away semi-important stuff like missing documentation tags on certain elements or if further down the line we introduce more logic / code - the same pragmas will apply to the new code too, and we won't be able to notice those problems because of previously existing pragmas.

Reply:
Removed.